eng/common/tools.ps1

```diff
@@ -168,6 +168,10 @@ function InitializeDotNetCli([bool]$install, [bool]$createSdkLocationFile) {
     $env:DOTNET_CLI_TELEMETRY_OPTOUT=1
   }
 
+  # Enable MSBuild server for faster builds
+  $env:DOTNET_CLI_USE_MSBUILD_SERVER=1
+  $env:MSBUILDUSESERVER=1
```

Comment:
Modifying `eng/common` won't work long term - it will be overridden back on next arcade update.

Have you considered modifying arcade itself so all .net repos dogfood this?